src/PostLMButton.ts

```diff
@@ -0,0 +1,51 @@
+import {Style} from "./Style";
+
+export class PostLMButton {
```

Comment:
Wouldn't a `implements Module` be good here? I just saw it's also missing in the other modules.

Reply:
Its an implicit interface. If one of the modules doesn't fit, the thing that accepts them will complain. This means that the individual modules aren't tied to a specific interface, but everything is still typesafe from the point of view of the consumer. This is why the module interface is also private